thorlcr/slave/thslavemain.cpp

```diff
@@ -85,7 +85,7 @@ static void replyError(unsigned errorCode, const char *errorMsg)
     Owned<IException> e = MakeStringException(errorCode, "%s", str.str());
     CMessageBuffer msg;
     serializeException(e, msg);
-    queryClusterComm().send(msg, 0, MPTAG_THORREGISTRATION);
+    queryWorldCommunicator().send(msg, 0, MPTAG_THORREGISTRATION);
```

Comment:
the world comm, can receive from anyone, the nodecomm can receive from only the thorgroup..
So could be nodecomm().

The idea originally was others could dynamically register, so needed to list to 'world'.
I changed it to queryWorldCommunicator() because it matched what slave was using, which doesn't know the group yet..